src/settings-ui/Settings.UI/GlobalSuppressions.cs

```diff
@@ -73,3 +73,8 @@
 [assembly: SuppressMessage("StyleCop.CSharp.MaintainabilityRules", "SA1402:File may only contain a single type", Justification = "Not part of the project. WindowsAppSDK file.", Scope = "type", Target = "~T:Microsoft.WindowsAppSDK.Runtime.Packages.Singleton")]
 [assembly: SuppressMessage("StyleCop.CSharp.LayoutRules", "SA1516:Elements should be separated by blank line", Justification = "Not part of the project. WindowsAppSDK file.", Scope = "type", Target = "~T:Microsoft.WindowsAppSDK.Runtime.Packages.Singleton")]
 [assembly: SuppressMessage("StyleCop.CSharp.LayoutRules", "SA1516:Elements should be separated by blank line", Justification = "Not part of the project. WindowsAppSDK file.", Scope = "type", Target = "~T:Microsoft.WindowsAppSDK.Runtime.Packages.Main")]
+[assembly: SuppressMessage("StyleCop.CSharp.LayoutRules", "SA1513:Closing brace should be followed by blank line", Justification = "Not part of the project. WindowsAppSDK file.", Scope = "namespaceanddescendants", Target = "Microsoft.WindowsAppSDK.Runtime.Packages")]
```

Comment:
These are pretty important rules. Can we maybe target the offending file or method?

Reply:
I wouldn’t block the release on this but would be nice to adjust

Reply:
The target part means these exceptions will be applied to the offending package. They'll still trigger in our code.
Regarding the copyright one, I think that could only be done locally in the file for it to be specific ... It's code from nuget, though. It's possible to make that one only apply to the Settings project, so I guess that's what we'll do.